<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSetupApp</class>
 <widget class="QWidget" name="PageSetupApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Serial Port #2 (Powe Supply)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="selectBox_2">
        <property name="title">
         <string>Select Serial Port</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="8" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="manufacturerLabel_2">
           <property name="text">
            <string>Manufacturer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnScan_2">
           <property name="text">
            <string>Scan ports</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="vidLabel_2">
           <property name="text">
            <string>Vendor ID:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="serialNumberLabel_2">
           <property name="text">
            <string>Serial number:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="serialPortInfoListBox_2"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="locationLabel_2">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="descriptionLabel_2">
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="pidLabel_2">
           <property name="text">
            <string>Product ID:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="parametersBox_2">
        <property name="title">
         <string>Select Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="dataBitsLabel_2">
           <property name="text">
            <string>Data bits:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="parityLabel_2">
           <property name="text">
            <string>Parity:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="baudRateBox_2"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="parityBox_2"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="baudRateLabel_2">
           <property name="text">
            <string>BaudRate:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="dataBitsBox_2"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="stopBitsLabel_2">
           <property name="text">
            <string>Stop bits:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="stopBitsBox_2"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="flowControlLabel_2">
           <property name="text">
            <string>Flow control:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="flowControlBox_2"/>
         </item>
         <item row="5" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Serial Port #1 (Fluke)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="_1">
        <property name="title">
         <string>Select Serial Port</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="8" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="manufacturerLabel_1">
           <property name="text">
            <string>Manufacturer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnScan_1">
           <property name="text">
            <string>Scan ports</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="vidLabel_1">
           <property name="text">
            <string>Vendor ID:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="serialNumberLabel_1">
           <property name="text">
            <string>Serial number:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="serialPortInfoListBox_1"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="locationLabel_1">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="descriptionLabel_1">
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="pidLabel_1">
           <property name="text">
            <string>Product ID:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="parametersBox_1">
        <property name="title">
         <string>Select Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="dataBitsLabel_1">
           <property name="text">
            <string>Data bits:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="parityLabel_1">
           <property name="text">
            <string>Parity:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="baudRateBox_1"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="parityBox_1"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="baudRateLabel_1">
           <property name="text">
            <string>BaudRate:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="dataBitsBox_1"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="stopBitsLabel_1">
           <property name="text">
            <string>Stop bits:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="stopBitsBox_1"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="flowControlLabel_1">
           <property name="text">
            <string>Flow control:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="flowControlBox_1"/>
         </item>
         <item row="5" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QPushButton" name="btnApply">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
